lib/iris/cube.py

```diff
@@ -3432,7 +3431,7 @@ def regrid(self, grid, scheme):
         return regridder(self)
 
 
-class ClassDict(object, UserDict.DictMixin):
+class ClassDict(collections.MutableMapping, object):
```

Comment:
I'm not sure why it's there; I know I had to swap the order to get the MRO to work. It looks to have existed since the initial commit by @rhattersley.

That being said, `MutableMapping` is only an interface, whereas `object` is the whole new-style class type.